__tests__/app.test.js

```diff
@@ -162,14 +173,93 @@ describe("/api/articles/:article_id/comments", () => {
         expect(body.message).toBe("ID not found");
       });
   });
-  test("Get 200: responds with empty array when article_id is valid but no comments found ", () => {
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("POST 201: add a comment to given article comments and respond with newly created comment", () => {
     return request(app)
-      .get("/api/articles/2/comments")
-      .expect(200)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "icellusedkars",
+        body: "Am I the batman?",
+      })
+      .expect(201)
       .then(({ body }) => {
-        const { comments } = body;
-        expect(comments).toBeInstanceOf(Array);
-        expect(comments).toHaveLength(0);
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: "Am I the batman?",
+          article_id: 1,
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("Post 201: add a comment to given article comments ignoring extra keys in input body", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "icellusedkars",
+        body: "too many keys",
+        rankdomKey: 12,
+      })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toMatchObject({
+          comment_id: expect.any(Number),
+          body: "too many keys",
+          article_id: 1,
+          author: expect.any(String),
+          votes: expect.any(Number),
+          created_at: expect.any(String),
+        });
+      });
+  });
+  test("Status 400: reposnds with an error message when missing required information", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({
+        username: "user1234",
```

Comment:
This test should use a valid username, as you are testing for different functionality here.